src/particles/diagnostics/ReducedBeamCharacteristics.cpp

```diff
@@ -0,0 +1,251 @@
+/* Copyright 2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Marco Garten, Chad Mitchell, Yinjian Zhao, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+
+#include "ImpactX.H"
+#include "particles/ImpactXParticleContainer.H"
+#include "particles/diagnostics/ReducedBeamCharacteristics.H"
```

Comment:
Own import always first :)
```suggestion
#include "ReducedBeamCharacteristics.H"

#include "ImpactX.H"
#include "particles/ImpactXParticleContainer.H"
```